src/placement/jump_map_versions.c

```diff
@@ -886,7 +886,7 @@ get_target(struct pool_domain *root, struct pool_domain *curr_pd, uint32_t layou
 
 		get_target_v0(root, target, key, dom_used, dom_full, dom_cur_grp_used,
 			      tgts_used, shard_num, allow_status, fdom_lvl);
-		if (spare_avail) {
+		if (spare_avail && spare_left) {
```

Comment:
if spare_avail is not NULL, but spare_left is NULL, spare_avail will be unknow is that expected?

Reply:
actually both of them should be NULL or not NULL, but clearly Coverity does not like it.